PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-aad/src/main/java/com/microsoft/azure/toolkit/intellij/connector/aad/AzureApplicationComboBox.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.connector.aad;
+
+import com.microsoft.azure.toolkit.intellij.common.AzureComboBox;
+import com.microsoft.graph.models.Application;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Combobox listing Azure AD applications.
+ */
+class AzureApplicationComboBox extends AzureComboBox<Application> {
+    private final Supplier<List<Application>> supplier;
```

Comment:
no need to add this field, 
you can use `get/setItemsLoader` of `AzureComboBox` directly.

Reply:
Thank you, that's a good point! I missed the superclass's item loader. I've updated this PR to use it instead.